services/dy-3dvis/docker/custom/entrypoint.bash

```diff
@@ -36,40 +36,34 @@ fi
 # try to pull data from S3
 echo
 echo "trying to download previous state..."
-python docker/state_manager.py pull --path ${SIMCORE_NODE_APP_STATE_PATH} --silent
+python utils/state_manager.py pull --path "${SIMCORE_NODE_APP_STATE_PATH}" --silent
```

Comment:
I would like to suggest a small change here and check whether a path is actually defined before pulling
```bash
if [ -n "${SIMCORE_NODE_BASEPATH}" ]; then
  echo "$INFO" "Restoring previous state..."
  python /utils/state_manager.py "${SIMCORE_NODE_APP_STATE_PATH}" --silent
else
  echo "$WARNING" "SIMCORE_NODE_APP_STATE_PATH was not set. Saving states feature is disabled."
fi
```

Reply:
you mean in the sense of a service where there is no state?
but in this case it has to be defined or else something is wrong. 

Maybe we need something like a has_state flag or so?

Reply:
I needed to add this to start a service that had no state stored (e.g. when starting it locally) 
In any case, as you know, all this logic should go in the sidecar.